code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/components/SmallImageCard.kt

```diff
@@ -54,7 +54,7 @@ fun SmallImageCard(
     Card(
         modifier = Modifier
             .clickable {
-                observer?.onEvent(UIEvent.Interact(ui, UIAction.CLICK))
+                observer?.onEvent(UIEvent.Interact(ui, UIAction.Click(null, ui.getTemplate().actionUrl)))
```

Comment:
nit: `SmallImageTemplate` has an id (same as propositionId). Do you think it would be useful to pass that here?

Reply:
we can pass it here for ease of implementation